### PR TITLE
Add taxonomy to template

### PR DESCRIPTION
**Pull request by BarryPiccinni**
Add taxonomy to template and include the `classification` key in each schema.

### CODE REVIEW

**Review by armoj — CHANGES_REQUESTED** [review]
Can we have a group discussion around if this is a minor or a patch.

`MINOR version when you add functionality to the API in a backwards compatible manner` - definition fits this PR.

`PATCH version when you make backwards compatible bug fixes or other changes not affecting the API.` - "or other changes" fits this PR also

File: src/lib/resource/sections/applicant-are-you-18-or-over.js [hunk 1]
```diff
@@ -14,7 +14,12 @@ module.exports = {
                     oneOf: [
                         {title: 'Yes', const: true},
                         {title: 'No', const: false}
-                    ]
+                    ],
+                    meta: {
+                        classifications: {
+                            theme: "about-application",
```
[the remark this answers is not in the document]
Comment (armoj) [reply src/lib/resource/sections/applicant-are-you-18-or-over.js:20]: how did the linter not pick this up? Hmmm

File: src/lib/resource/sections/applicant-are-you-registered-with-gp.js [hunk 2]
```diff
@@ -14,7 +14,12 @@ module.exports = {
                     oneOf: [
                         {title: 'Yes', const: true},
                         {title: 'No', const: false}
-                    ]
+                    ],
+                    meta: {
+                        classifications: {
+                            theme: "treatment",
```
Comment (neil-stephen-mcgonigle) [inline src/lib/resource/sections/applicant-are-you-registered-with-gp.js:20]: single quotes

File: src/lib/resource/sections/applicant-are-you-claiming-for-physical-injuries.js [hunk 3]
```diff
@@ -14,7 +14,12 @@ module.exports = {
                     oneOf: [
                         {title: 'Yes', const: true},
                         {title: 'No', const: false}
-                    ]
+                    ],
+                    meta: {
+                        classifications: {
+                            theme: "injuries",
```
Comment (neil-stephen-mcgonigle) [inline src/lib/resource/sections/applicant-are-you-claiming-for-physical-injuries.js:20]: single quotes

**Pull request closed without merging**